### PR TITLE
[Network] `az network public-ip create`: Fix the issue when using `--ip-tags`

### PR DESCRIPTION
**Pull request by ZengTaoxu**
**Related command**

az network public-ip create

**Description**

Related IcM: https://portal.microsofticm.com/imp/v3/incidents/details/350795563
image

**Testing Guide**

azdev test test_network_public_ip

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Network

**Review by necusjz — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/network/_validators.py [hunk 1]
```diff
@@ -482,7 +482,7 @@ def validate_ip_tags(cmd, namespace):
         ip_tags = []
         for item in namespace.ip_tags:
             tag_type, tag_value = item.split('=', 1)
-            ip_tags.append(IpTag(ip_tag_type=tag_type, tag=tag_value))
+            ip_tags.append({"ip_tag_type": tag_type, "tag": tag_value})
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/_validators.py:485]: please remove `IpTag = cmd.get_models('IpTag')`

Comment (kairu-ms) [reply src/azure-cli/azure/cli/command_modules/network/_validators.py:485]: It's better to rewrite this command by inherent way.

**Pull request merged**